release-notes.txt

```diff
@@ -1,3 +1,548 @@
+- remove support for wsn430-1_3b and wsn430-1_4
+
+CPU
+---
+* cortexm:
+    + Make no_thread_idle compatible with cortex-m0
+    + Add thread info to hard fault handler
+    + advertise puf_sram feature
+    * Skip context save/restore on scheduling the previous thread
+    * use mpu stack guard if DEVELHELP is enabled
+    * fix check for bitbanding
+    * Flush pipeline after triggering PendSV
++ cc2538: add Watchdog implementation
+* fe310:
+    + Add custom PLIC driver as peripheral
+    * Merge and optimize trap_entry with irq_arch
+    * Unify linker scripts
+* msp430: Update to inline-able IRQ API Area
+* nrf5x_common: UART: add support for 1 MBaud
+* stm32l0l1: rework clock initialization and configuration
+* stm32:
+    + generate vectors C files and IRQ defines from CMSIS headers
+    + stm32cmsis: retrieve STM32 CMSIS header from a package
+    + g0: add basic support
+    + implement periph_timer_periodic
+    * Add support for arbitrary SPI clock rate
+    * f3: fix clock configuration for HSI
+    * l0: rework clock initialization and configuration
+    * l1: rework clock initialization and configuration
+    * l4: rework clock configuration and initialization
+    * wb: rework clock configuration and initialization
+    - disable MPU for cortex-m0+
+* sam0:
+    + GPIO: use tamper detection to wake from Deep Sleep
+    + add support to run PWM off TC timers
+    + ADC: add support for samd5x/same5x
+    + Add support for Single-cycle I/O Port for GPIO
+    * UART: implement arithmetic BAUD mode
+    * merge RTC & RTT implementation
+    * flashpage: clean up implementation
+- saml1x: Remove MPU feature
+
+Device Drivers
+--------------
++ at86rf2xx: spi security module
++ at86rf2xx: support frame retry reporting on AT86RFR2
++ soft_uart: add software based UART implementation
++ scd30 : Add driver for scd30 sensor
++ stm32_eth: add 'NETDEV_EVENT_LINK_UP' event
++ Add support for Adafruit Seesaw Soil moisture sensor
++ radio_hal:
+    + introduce Radio HAL for IEEE802.15.4 compatible radios
+    + cc2538_rf: implement Radio HAL
+    + nrf52840: implement Radio HAL
++ mtd: add page addressed operation
+* stmpe811: fix interrupt polarity & interrupt generation
+* sps30: Update units of measurement
+* sps30: Add sleep mode
+- sdcard_spi: remove auto-init
+
+Build System / Tooling
+----------------------
++ Add libstdcpp feature
++ gnrc_border_router: Add USB Ethernet support
++ buspirate: add Bus Pirate as Debug Adapter
+* README: Add more badges
+* Convert remaining Python2 users to Python3
+* openocd-adapters: select transport based on adapter
+* Makefile.{base,include}: Fix linking for C++ code in external modules
+* link object files instead of archives
+* tapsetup:
+
+    - Add support for listing bridges
+
+    - Add support to configure addresses for bridge
+
+    - Add support for configuring routes and forwarding capabilities for bridge
+
+    - Add TUN support
```

Comment:
Something happened here when moving from the etherpad :-/

Reply:
Thanks! Amended with less newlines